admin/Makefile

```
@@ -1,80 +1,36 @@
-BUILD_DIR ?= .build
-VENV ?= ${BUILD_DIR}/venv
-DOCKER_TAG ?= latest
-
-GIT_HASH := $(shell git rev-parse HEAD)
-DOCKER_BASE = camptocamp/c2cgeoportal
+# Used for development purpose: please do not remove this file
 
-.PHONY: build
-build: ${BUILD_DIR}/requirements.timestamp ${BUILD_DIR}/node_modules.timestamp
-
-.PHONY: clean
-clean:
-	rm -rf ${BUILD_DIR}/*.timestamp
-
-.PHONY: cleanall
-cleanall:
-	rm -rf ${BUILD_DIR}
-	rm -rf node_modules
+DOCKER_BASE ?= camptocamp/c2cgeoportal
+DOCKER_TAG ?= latest
 
-.PHONY: test
-test: build ${BUILD_DIR}/requirements-dev.timestamp
-	# Run your project's tests.
-	${VENV}/bin/pytest --driver Chrome
+default: help
 
-.PHONY: check
-check: flake8
+.PHONY: help
+help: ## Display this help message
+	@echo "Usage: make <target>"
+	@echo
+	@echo "Possible targets:"
+	@grep -Eh '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "    %-20s%s\n", $$1, $$2}'
 
-.PHONY: flake8
-flake8: ${BUILD_DIR}/requirements-dev.timestamp
-	${VENV}/bin/flake8 c2cgeoportal_admin acceptance_tests *.py
+.PHONY: preparedev
+preparedev: ## Run this once before serve
+	make -C .. build-tools build-test-db
```

```suggestion
	make -C .. build-runner build-test-db
```
Or the base image in the Dockerfile is wrong...